vdev/src/testing/runner.rs

```diff
@@ -80,8 +80,13 @@ pub fn get_agent_test_runner(container: bool) -> Result<Box<dyn TestRunner>> {
 }
 
 pub trait TestRunner {
-    fn test(&self, outer_env: &Environment, inner_env: &Environment, args: &[String])
-        -> Result<()>;
+    fn test(
+        &self,
+        outer_env: &Environment,
+        inner_env: &Environment,
+        features: Option<&[String]>,
```

Comment:
I'm not seeing this new parameter in the trait actually being used anywhere. Did I miss it?

Reply:
It's used in this file down on line 270, for the `ContainerTestRunner` implementation of the trait. (which is called from `IntegrationTest::test()`